api/src/main/java/javax/mvc/MvcContext.java

```diff
@@ -123,4 +125,12 @@
      * @return instance of encoders.
      */
     Encoders getEncoders();
+
+    /**
+     * Returns the locale of the current request.
+     *
+     * @return The request locale
+     */
```

Comment:
Yeah, I think Ivar is correct. Without that sentence you will get an empty text in the method table in the javadocs. So I would prefer to keep it.